mantidimaging/gui/windows/spectrum_viewer/presenter.py

```diff
@@ -305,6 +312,11 @@ def handle_enable_normalised(self, enabled: bool) -> None:
             self.spectrum_mode = SpecType.SAMPLE
         self.redraw_all_rois()
         self.view.display_normalise_error()
+        averaged_image = self.model.get_normalized_averaged_image() if enabled else self.model.get_averaged_image()
+        if averaged_image is None:
+            image_shape = self.model.get_image_shape()
+            averaged_image = np.zeros(image_shape, dtype=np.float32)
+        self.view.set_image(averaged_image, autoLevels=True)
```

Comment:
Can this be replaced with a call to `self.update_displayed_image(autoLevels=True)`